### DISCUSSION TITLE
How to bind the text of tooltip ?

### DISCUSSION THREAD

**Discussion by MohAid** [zauberzeug/nicegui#2264] [Q&A] [answered]
### Question

I am trying to bind the text in the tooltip of an element but I failed to do that in many ways, any idea, please?

**Comment by falkoschindler** [comment, marked as answer]
You can use a nested `ui.tooltip()` element, which has a bindable `text` property:

```py
tooltip_input = ui.input(value='This is a tooltip')

with ui.label('This label has a tooltip'):
    ui.tooltip().bind_text_from(tooltip_input, 'value')
```

**Reply by MohAid** [reply]
It seems I must (not can!) :)
thank you @falkoschindler. So, in this case, it has to be nested to handle it as a different element (as is it originally), and it is not able to be handled (or bound) when I create it in: `ui.button('xxxx').tooltip('xxxx')` do you agree with that?

**Reply by falkoschindler** [reply]
Exactly. 👍🏻
